src/styles.css

```diff
@@ -0,0 +1,117 @@
+/* styles.css */
```

Comment:
Looks like there's some styles in `index.css` that are being overridden by the styles here.